### DISCUSSION TITLE
Możliwość listowania ofert i zakupu

### DISCUSSION THREAD

**Discussion by Arputikos** [allegro/allegro-api#6813] [Suggestions - rejected]
Witam,
Szukałem ale nie znalazłem takiej informacji - czy obecne api oferuje możliwość wyszukowania i filtrowania ofert, oraz zakupu wybranej oferty? Wybrania adresu dostawy, płatności itp? Albo chociaż jakakolwiek możliwość zakupu z poziomu API?
Dziękuję

**Comment by AureliuszBrussy** [comment]
Jako, że w zakładce [Discussions](https://github.com/allegro/allegro-api/discussions) wg. naszych założeń mają się znajdował sugestię, zamykam tutaj temat i przenoszę go do [Issues](https://github.com/allegro/allegro-api/issues).
